xmake/rules/c++/modules/modules_support/clang.lua

```diff
@@ -298,7 +298,7 @@ function generate_dependencies(target, sourcebatch, opt)
                         end
                     end
 
-                    assert(not (has_std_modules and target:values("c++.clang.modules.strict")),
+                    assert(not (has_std_modules and not target:values("c++.clang.modules.stdmodules")),
```

Comment:
maybe we use `set_policy("build.c++.clang.modules.stdmodules", true)` and target:policy() will be better.

Reply:
> error: unknown policy(c++.clang.modules.stdmodules)!
policies are declared where ?